### DISCUSSION TITLE
Unable to create additional column under page

### DISCUSSION THREAD

**Discussion by UmerTariqKhan** [mixcore/mix.core#455] [Q&A] [answered]
1- Unable to create additional columns under Page although they are created in DB but not showing in page.
2- Secondly When I try to create additional columns under Module (Its replica) in another language it says "name already exists". it forces me to create a column with a new name.

Any help will be appreciated.
Thanks

1-
![image](https://user-images.githubusercontent.com/36357077/128264819-10587816-8f53-457e-9fbd-f3e8daa6c84e.png)
![image](https://user-images.githubusercontent.com/36357077/128264988-58aa5b4f-afc3-4ad0-93af-c1a54cf43c21.png)

2-
![image](https://user-images.githubusercontent.com/36357077/128265278-3d96cf0a-6271-4d36-a4c5-a5ce7413d3d8.png)

**Comment by nhathoang989** [comment]
Hello, @UmerTariqKhan.

1.  I think because you did not save the page before leaving, thus the navigation between the page and the column was lost.

=> However, if the page has already been saved before, I included the feature auto save the navigation in this commit https://github.com/mixcore/mix.core/pull/456/commits/4a103f2403289f6dc0a4b1f11b73788282bd2b49

2. I'm unable to reproduce your issue.

**Comment by UmerTariqKhan** [comment]
Thanks for your support.
I am still unable to create columns under Page, I delete columns manually
from [dbo].[mix_database_column]. Please see attached clip file. Thanks &
Regards

https://drive.google.com/file/d/1xA4jRWOtfnz7fQif8OuQ2GAG9xJ9u84b/view?usp=sharing

**Comment by nhathoang989** [comment, marked as answer]
There's an issue about additional data.
I updated in this PR and merged to master https://github.com/mixcore/mix.core/pull/457
Thanks for your reporting.

**Comment by UmerTariqKhan** [comment]
Hi, I fetch code now and found it didn't fix yet?

My Post Type  = PostList

**Comment by UmerTariqKhan** [comment]
Under page, MixDataBase is saving as "sys_additional_column". I think this
is the issue?

**Reply by nhathoang989** [reply]
I don't think so. Did you inspect browser to check if there is any failed request when save the page?
Btw, can you deploy your code to the testing server that you shared in previous discussion.

**Comment by UmerTariqKhan** [comment]
let me check and let you know

**Comment by UmerTariqKhan** [comment]
Thanks. Now I am able to create columns, but columns are created with
status = Deleted. I filtered it with all status then I am able to see
crerated columns.

**Comment by UmerTariqKhan** [comment]
Thanks a lot.
Please let me know any info about the issue #458
https://github.com/mixcore/mix.core/issues/458. I am working twice first in
English and then in Arabic. Please let me know. Thanks so much.

**Comment by UmerTariqKhan** [comment]
Hi, Now I am unable to create Columns under Post, I am trying to create
columns under Page>Post, but the add column link is doing nothing, Nothing
is happening on Add Column link, the latest release is deployed on the link
http://54.159.73.158:8081/.
Please help I am stuck.

Thanks and regards
***@***.***

**Reply by nhathoang989** [reply]
**1. Your old post has the posttype = "undefined" in the test server, thus it can't load the new data.**

![image](https://user-images.githubusercontent.com/5327699/128437147-01ea4c9d-eb57-4b89-b347-ffc2b69ce1d9.png)


**==> You can select again to update the posttype.**


![image](https://user-images.githubusercontent.com/5327699/128437430-49a3877a-16ed-459b-ad6c-6279697cd07f.png)


**2. If all of your pages/posts/modules have the same extra columns, you may add them to the default database below, and they'll be added automatically when you create a new one.**


![image](https://user-images.githubusercontent.com/5327699/128440517-f95cd1df-f071-4330-9b8e-74f56756248a.png)


**3. Please download the most recent source code; I was trying to cover a few additional scenarios when upgrading from previous versions.**

**Reply by nhathoang989** [reply]
Btw, my skype is nhathoang989, you can contact me if need

**Comment by UmerTariqKhan** [comment]
Thanks, soo much
